libsolc/libsolc.cpp

```diff
@@ -229,12 +229,12 @@ string compile(StringMap const& _sources, bool _optimize, CStyleReadFileCallback
 
 string compileMulti(string const& _input, bool _optimize, CStyleReadFileCallback _readCallback = nullptr)
 {
-	Json::Reader reader;
+	std::string errs;
 	Json::Value input;
-	if (!reader.parse(_input, input, false))
+	if (!dev::jsonParseStrict(_input, &input, &errs))
```

Comment:
The `dev` namespace is imported in this file.

Reply:
Yop.